src/Components/Individual-product.js

```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import BottleImg from "./bottleImg";
 import './comp.css';
 import Description from "./Description";
-function Individual_product() {
+function Individual_product(props) {
```

Comment:
The naming convention of the component has to be fixed. Also, the file name has to match that naming convention.